visualize.Rmd

```diff
@@ -1,115 +1,113 @@
-# Data visualisation
+# Visualización de datos
 
-## Introduction
+## Introducción
 
-> "The simple graph has brought more information to the data analyst’s mind 
-> than any other device." --- John Tukey
+>"Un simple gráfico brinda más información a la mente del analista de datos
+>que cualquier otro dispositivo". --- John Tukey
 
-This chapter will teach you how to visualise your data using ggplot2. R has several systems for making graphs, but ggplot2 is one of the most elegant and most versatile. ggplot2 implements the __grammar of graphics__, a coherent system for describing and building graphs. With ggplot2, you can do more faster by learning one system and applying it in many places.
+En este capítulo aprenderás cómo visualizar datos usando el paquete **ggplot2**. De los muchos sistemas que posee R para hacer gráficos, **ggplot2** es uno de los más elegantes y versátiles. Esto se debe a que implementa  un sistema coherente para describir y construir gráficos, conocido como la __gramática de gráficos__. Con **ggplot2** puedes hacer más cosas en menor tiempo, aprendiendo un único sistema y aplicándolo en muchos lugares
```

Comment:
* Podría utilizarse *en diferentes ámbitos* como traducción de *in many places*.

Reply:
De acuerdo con cambiar *en muchos lugares* por *en diferentes ámbitos*. Falta también un punto al final de la frase.